server/functional-test/src/main/java/org/zanata/page/AbstractPage.java

```diff
@@ -22,6 +22,8 @@
 
 import java.util.List;
 import java.util.Set;
+
+import org.assertj.core.util.Lists;
```

Comment:
There's no real harm in using assertj's Lists class, but if you get in the habit of using, say, the Lists class from Guava, you can use it any code (eg non-test code, client code). Or if an immutable list is all you need, you can use Guava's ImmutableList.of(...) method.